lib/tomo/plugin/puma/tasks.rb

```diff
@@ -38,6 +38,10 @@ def log
       remote.attach "journalctl", "-q", raw("--user-unit=#{service.name.shellescape}"), *settings[:run_args]
     end
 
+    def tail_log
+      remote.run "journalctl -q --user-unit=#{service.name.shellescape} -f"
```

Comment:
I think this should be `attach`, because `-f` establishes a long-running process; no further tomo commands would be possible after this.

```suggestion
      remote.attach "journalctl -q --user-unit=#{service.name.shellescape} -f"
```